drivers/volume/portworx/portworx.go

```diff
@@ -2340,25 +2340,14 @@ func (d *portworx) getNodeManagerByAddress(addr string) (api.OpenStorageNodeClie
 
 func (d *portworx) maintenanceOp(n node.Node, op string) error {
 	var err error
-	// TODO replace by sdk call whenever it is available
-	pxdRestPort, err := getRestPort()
+
+	logrus.Warnf("unable to get service endpoint falling back to node addr: err=%v, skipPXSvcEndpoint=%v", err, d.skipPXSvcEndpoint)
```

Comment:
this warning is misleading; need to reword it

also, add a comment about why we cannot use the service endpoint here

Reply:
updated